src/detours.cpp

```diff
@@ -1395,6 +1395,12 @@ PVOID WINAPI DetourAllocateRegionWithinJumpBounds(_In_ LPCVOID pbTarget,
     return pbNewlyAllocated;
 }
 
+BOOL WINAPI DetourIsFunctionImported(_In_ PBYTE pbCode,
```

Comment:
It would be nice if you could add new tests for `DetourIsFunctionImported` as it's now exposing public functionality. I would suggest some unit tests under [tests/](https://github.com/microsoft/Detours/tree/master/tests).

Reply:
I've been trying to find something to do this without just pulling a value out of the IAT, but is it possible to write some expression to get the address of the address table entry in C/C++?  My expectation is that it is mostly a hidden implementation detail, but hopefully there is some way of getting it using MSVC specifics.

```
// Generates load from IAT rather than being address of IAT
auto ptr = &SetLastError;

// If only it were this easy
auto IAT_ptr = &&SetLastError;
```

Reply:
```cpp
// Get the address of static imports, w/o relying on __declspec(dllimport)
// or LoadLibrary/GetProcAddress. Consider using decltype(&foo) __imp_foo.
#ifdef _X86_
#pragma warning(disable:4483) // disable warning/error about __identifier(<string>)
#define __imp_DbgPrint                          _imp__DbgPrint
#define __imp_LoadLibraryW                   __identifier("_imp__LoadLibraryW@4")
#endif

extern size_t const __imp_DbgPrint;
extern size_t const __imp_LoadLibraryW;
```
etc.